src/spatial/reaction_rates.jl

```diff
@@ -52,13 +52,20 @@ end
 
 update rates of all reactions in rxs at site
 """
-function update_rx_rates!(rx_rates, rxs, u, site)
+function update_rx_rates!(rx_rates::RxRates{F,M}, rxs, u, site) where {F, M <: MassActionJump}
```

Comment:
Why do we still need this version? Shouldn't we always just `SpatialMassActionJump`s internally for  the spatial solvers now?

Reply:
`MassActionJump` has more utilities like parameter mapping and merging so I thought it's good to leave it as a sort of backward compatibility